config/routes.rb

```diff
@@ -3,6 +3,8 @@
 
   # Reveal health status on /up that returns 200 if the app boots with no exceptions, otherwise 500.
   # Can be used by load balancers and uptime monitors to verify that the app is live.
+  delete'/events/:id',  to: 'events#destroy'
```

Comment:
This also looks good but. Youre doing mannually what resources do.
[Resources](https://guides.rubyonrails.org/routing.html#singular-resources)
[ANother reading](https://launchschool.com/books/demystifying_rails/read/routes_and_resources)